osu.Game/Rulesets/UI/DrawableRuleset.cs

```diff
@@ -453,6 +458,8 @@ public HitWindows FirstAvailableHitWindows
         /// <param name="continueResume">The action to run when resuming is to be completed.</param>
         public abstract void RequestResume(Action continueResume);
 
+        public abstract void CancelResume();
```

Comment:
This addition is going to be a part of the public API surface, since custom rulesets might want to override this, so an xmldoc here is probably necessary.